### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by dakahn** [carbon-design-system/carbon-website#294]
### Steps to reproduce:
1) Install aXe browser extension for Chrome or Firefox
2) Run an audit on website

### Issues:
1) Search input has no label - Critical
_There is a label, but it's hidden_
2) Search input has inappropriate aria role - Minor
_The search bar has a role of combobox when a role of search on a landmark form container would be more helpful_
3) Main page content is not contained in a landmark - Moderate
4) Chevron button on side nav is a 2.49:1 contrast ratio 
_This ratio is failing AA/AAA WCAG Accessibiltiy guidelines for contrast_

**Comment by dakahn** [comment]
Doing some audits. Happy to fix the bugs I found. 😺

**Comment by dakahn** [comment]
Roger that.
